maven-java/kalix-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/README.md

```diff
@@ -25,22 +25,20 @@ mvn compile
 #[[
 ## Running Locally
 ]]#
-In order to run your application locally, you must run the Kalix proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
-To start the proxy, run the following command from this directory:
 
-```shell
-docker-compose up
-```
-
-> Note: if you're looking to use Google Pub/Sub, see comments inside [docker-compose.yml](./docker-compose.yml) 
-> on how to enable a Google Pub/Sub emulator that Kalix proxy will connect to.
+When running a Kalix application locally, at least two applications are required. The current Kalix application and its companion Kalix Proxy.
```

Comment:
I find this phrase where we use the term "application" with different meanings a bit confusing.. 

```suggestion
When running a Kalix service locally, we need to have its companion Kalix Proxy running alongside.
```

Reply:
Totally agree.